Source/WebCore/page/scrolling/ScrollingTree.cpp

```diff
@@ -517,6 +517,33 @@ void ScrollingTree::clearLatchedNode()
     m_latchingController.clearLatchedNode();
 }
 
+void ScrollingTree::receivedWheelEvent(const PlatformWheelEvent& event)
+{
+    auto monitor = m_wheelEventTestMonitor;
+    if (monitor)
+        monitor->receivedWheelEvent(event);
```

Comment:
I know this isn't new, but I'm not sure that the temporary variable buys us anything here. I would probably just use the member.